logging/Limiting_Access_to_Logs.md

```diff
@@ -15,101 +26,172 @@ only for a specified namespace.
 
-- Removes the Kibana tenant space
-- Removes the access controls for the Kibana tenant space
+1. Remove the Kibana-tenant space.
+2. Remove the access controls for the Kibana-tenant space.
 
 The `offboard.sh` script does not remove the initial user, if one was 
 created by the `onboard.sh` script.
 
-This is the syntax for the script:
+Here is the syntax for the script:
 
-```
-/logging/bin/offboard.sh --namespace <namespace>  [--tenant <tenant>]
-```
-The `namespace` parameter is required. It specifies the Kubernetes namespace for which the corresponding Kibana tenant space and access controls will 
-be removed.
+<pre>
+/logging/bin/offboard.sh --namespace <i>namespace</i>  [--tenant <i>tenant</i>]
+</pre>
 
-The `tenant` parameter specifies the SAS Viya tenant for which the corresponding Kibana tenant space and access controls will 
-be removed.
+- *namespace* is required. It specifies the Kubernetes namespace for which the 
+  corresponding Kibana-tenant space and access controls are removed.
+- *tenant* specifies the SAS Viya tenant for which the corresponding 
+  Kibana-tenant space and access controls are removed.
 
-As with the `onboard.sh` script, the Kibana tenant space name is a 
-combination of the namespace name and the SAS Viya tenant name (if used). For example, if you ran the script and specified only `--namespace mynamespace`, the script would remove the Kibana tenant space named `mynamespace`. If you ran the script and specified `--namespace mynamespace --tenant mytenant1`, the script would remove the Kibana tenant space named `mynamespace_mytenant1`. 
+As with the `onboard.sh` script, the Kibana-tenant space name is a 
+combination of the namespace name and the SAS Viya tenant name (if used). For example, if you ran the script and specified only `--namespace mynamespace`, the script would remove the Kibana-tenant space named `mynamespace`. If you ran the script and specified 
+`--namespace mynamespace --tenant mytenant1`, the script would remove the Kibana-tenant space named `mynamespace_mytenant1`. 
 
 ## Managing User Accounts for Restricted Access
 
-Use the `/logging/bin/user.sh` script to create or delete Kibana accounts 
-that have access to only the log messages and Kibana tenant space 
-associated with a specified namespace or SAS Viya tenant. You can use 
-the script to create user accounts for a Kibana tenant space anytime 
-after you have run the `onboard.sh` to implement access limitations for 
-the Kibana tenant space.
+### Introduction
+
+You can create or delete Kibana accounts 
+that have access to only the log messages and Kibana-tenant space 
+associated with a specified namespace or SAS Viya tenant. 
 
-This is the syntax for the script to add a user:
+### Creating User Accounts
+After you run the `onboard.sh` to implement access limitations for 
+the Kibana-tenant space, you can run 
+the  `/logging/bin/user.sh` script to create user accounts for a Kibana-tenant space. 
```

Comment:
Hmmm,  maybe change: _"...create user accounts for a Kibana-tenant space."_ to _"...create user accounts that are bound by these access controls."_  The point is that these users are not only limited to the Kibana-tenant space.

Reply:
I made this adjustment. 
I also reflected the adjustment in the comment above for the same paragraph.